lib/Installation/Partitioner/FormattingOptionsPage.pm

```diff
@@ -88,7 +88,8 @@ sub select_filesystem {
     wait_screen_change(sub {
             send_key 'end';
     }, 20);
-    send_key_until_needlematch((sprintf FILESYSTEM_TYPE, $filesystem), 'up');
+    send_key 'alt-f';
```

Comment:
for what I can see in the failed job, swap is already selected, what about to skip this selection instead of clicking twice there?
btw we should not hardcode `alt-f`.

Reply:
it is not about swap. but /home. 
The thing is that the solution is expected to be as simple as the increase of the timeout. I am not sure about that. With the frequency of the broken test(for another reason) i cant really say. Reach me in RC if you want to discuss further.

Reply:
I can see udf partition with swap here https://openqa.suse.de/tests/4277506#step/msdos_partition_table/66
Please keep open this conversation so someone else can take a look, specially about that hardcoded `alt-`

Reply:
I think `send_key($self->{filesystem_shortcut});` already does exactly that. Also, keep in mind that this method is used in many other places, and we might have different shortcut for it too, so I would only act on `send_key_until_needlematch` call.